api.yaml

```diff
@@ -0,0 +1,65 @@
+swagger: "2.0"
```

Comment:
On reflection, I'm uncertain about including this.  It seems counter productive to include a swagger definition when we have reviewed swagger and decided it is not suitable.